src/app/universe/page.scss

```
@@ -0,0 +1,530 @@
+  }
+}
+
+.universe-shell {
+  position: absolute;
+  inset: 0;
+  isolation: isolate;
+  transition: filter 280ms ease;
+}
+
+.universe-shell.is-tuning {
+}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)